### DISCUSSION TITLE
Regarding Issue Templates

### DISCUSSION THREAD

**Discussion by mina-atef-00** [LesFruitsDefendus/saskatoon-ng#304] [Technical]
Could you follow [this guide](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-templates) to enable issue templates?
I believe you should choose `custom template`.

**Comment by tristanlatr** [comment]
Yes I'll follow this guide, I though I knew how to make it work but turns out I don't ^^

Thank you

**Comment by tristanlatr** [comment]
Done, sorry I must say that I've forgot about it.
